spikeextractors/sortingextractor.py

```diff
@@ -155,8 +165,13 @@ def get_unit_spike_features(self, unit_id, feature_name, start_frame=None, end_f
                             start_frame = 0
                         if end_frame is None:
                             end_frame = np.inf
-                        spike_indices =  np.where(np.logical_and(spike_train >= start_frame, spike_train < end_frame))
-                        return self._unit_features[unit_id][feature_name][spike_indices]
+                        if start_frame == 0 and end_frame == np.inf:
```

Comment:
Having inf for int is a bad idea. None sounds better. (or -1 but more dangerous)

Reply:
in this case it's fine because it's only used internally with the np.where (so -1 won't work)

Reply:
Yes it is fine but we have to avoid this everywhere. No inf for indexing.